package.json

```diff
@@ -32,7 +32,7 @@
     "@types/moment-timezone": "^0.5.12",
     "angular-froala-wysiwyg": "^3.0.0-rc.2-1",
     "core-js": "^2.4.1",
-    "materialize-css": "^0.97.8",
+    "materialize-css": "^1.0.0",
```

Comment:
Why this change here?

Reply:
i kind of don't know why they are here i am removing them sorry